core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/model/DroidKaigi2023Day.kt

```diff
@@ -81,5 +81,17 @@ public enum class DroidKaigi2023Day(
         public fun defaultDyamicThemeDate(): Boolean {
             return Day1.start < Clock.System.now()
         }
+
+        /**
+         * @return appropriate initial day for now
+         */
+        fun initialSelectedDay(): DroidKaigi2023Day {
+            val reversedEntries = entries.sortedByDescending { it.day }
```

Comment:
Thank you for refactoring! It worked as we expected! 👍

By the way, I came up with another solution:
```kotlin
fun initialSelectedDay(): DroidKaigi2023Day {
    val now = Clock.System.now()
    return ofOrNull(now) ?: Day1
}
```
The above one is not so accurate as your code since my code does not consider start and end time.
On the other hand, it looks a bit simpler 💡

Reply:
But your idea and implementation are so good 👍 I marked it as approved 💯